app/helpers/form_helper.rb

```diff
@@ -309,7 +318,7 @@ def link_to_add_fields(name, f, association, partial = nil, options = {})
   def field(f, attr, options = {})
     table_field = options.delete(:table_field)
     error       = options.delete(:error) || f.object.errors[attr] if f && f.object.respond_to?(:errors)
-    help_inline = help_inline(options.delete(:help_inline), error)
+    help_inline = help_inline(options.delete(:help_inline), error, {:indicator => options.delete(:indicator)})
```

Comment:
From looking [here](https://github.com/theforeman/foreman/pull/3594/files#diff-a1d6431830287bc2d6a5c744edbf1eb7R258), you have changed the syntax of `#help_inline` to pass the indicator as a first argument instead of an option.